domains/rmariuzzo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "GitHub profile page",
+  "repo": "https://github.com/rmariuzzo/rmariuzzo",
+  "owner": {
+    "username": "rmariuzzo",
+    "email": "rubens@mariuzzo.com"
+  },
+  "record": {
+    "CNAME": "https://github.com/rmariuzzo",
```

Comment:
CNAME has to be a host. If you have a [github pages subdomain](https://guides.github.com/features/pages/), I'd suggest you use that. Or you can just keep a url redirection.